### ISSUE TITLE
[Classic] Waterfox 2019.12: Corrupted Content Error on gmail is back

### ISSUE THREAD

**Issue by cobafesbuk** [MrAlex94/Waterfox#1331]
I just updated to to 2019.12 from 2019.10. This issue now is back.
My main reason to use Waterfox instead of Firefox Classic is because Waterfox fixed this issue.

Perhaps Google is doing something again to break gmail or maybe something was changed from 2019.10 to broke the fix?

![co](https://user-images.githubusercontent.com/4735940/71510656-281d5980-28c2-11ea-81fa-fd02c1897611.jpg)

**Comment by hawkeye116477** [comment]
Go to Preferences => Content and disable notifications, now should be good :smile:

**Comment by cobafesbuk** [comment]
After a bit Googling, I deleted gmail service worker yesterday and the error was gone because of that.

Thanks for your suggestion, I'll try it if the error surfaces again.

I can't reproduce the error anymore. Should I close this issue?

**Comment by cobafesbuk** [comment]
@hawkeye116477 
I tested your suggestion on my other computer, it's not working. 
I need to delete the gmail service worker to make the error gone.

**Comment by grahamperrin** [comment]
['Corrupted Content Error' while attempting to access Gmail · Issue #794 · MrAlex94/Waterfox](https://github.com/MrAlex94/Waterfox/issues/794)

**Comment by hawkeye116477** [comment]
Disabling notifications make workers disabled also, so that should be working if gmail wasn't opened before changing settings or you'll need additionally to unregister service worker.

**Comment by cobafesbuk** [comment]
@grahamperrin 
Strange, it's working from the first time I installed Waterfox until the previous 2019.10.. Same with my other computer, the error appeared as soon as I installed the current 2019.12. 
First time I used Waterfox was around April-May, maybe I already used version 56.2.8 back then.
So the latest version re-enabled service worker because many sites can't be opened?
Too bad Block Service Workers is no longer supported for Firefox anymore.

@hawkeye116477 
I see, so the worker was already installed and was not removed even after changing the setting.
Now I already disabled the notification and removed the worker, I hope it will never appear again.

Thanks all for your help :)
I'm closing this issue.
